{
    "type": "indicator",
    "spec_version": "2.1",
    "id": "indicator--780c7b72-edf7-49eb-9f65-9954214e57cb",
    "created_by_ref": "identity--748e6444-f073-4c50-b558-f49be8897a81",
    "created": "2021-03-05T09:15:00.000Z",
    "modified": "2021-03-05T20:39:00.000Z",
    "name": "CVE-2021-28034",
    "description": "An issue was discovered in the stack_dst crate before 0.6.1 for Rust. Because of the push_inner behavior, a double free can occur upon a val.clone() panic.",
    "indicator_types": [
        "compromised"
    ],
    "pattern": "[(software:cpe='cpe:2.3:a:stack_dst_project:stack_dst:*:*:*:*:*:rust:*:*')]",
    "pattern_type": "stix",
    "pattern_version": "2.1",
    "valid_from": "2021-03-05T09:15:00Z",
    "external_references": [
        {
            "source_name": "cve2stix",
            "description": "This object was created using cve2stix from the Signals Corps.",
            "url": "https://github.com/signalscorps/cve2stix"
        }
    ],
    "object_marking_refs": [
        "marking-definition--613f2e26-407d-48c7-9eca-b8e91df99dc9"
    ],
    "extensions": {
        "extension-definition--b463c449-d022-48b7-b464-3e9c7ec5cf16": {
            "all_cpe23uris": [
                "cpe:2.3:a:stack_dst_project:stack_dst:*:*:*:*:*:rust:*:*"
            ],
            "extension_type": "property-extension",
            "vulnerable_cpeid": [
                "cpe:2.3:a:stack_dst_project:stack_dst:*:*:*:*:*:rust:*:*"
            ]
        }
    }
}